server/src/test/java/org/bonitasoft/web/rest/server/api/bpm/flownode/ActivityVariableResourceTest.java

```diff
@@ -91,36 +91,35 @@ public void should_return() throws DataNotFoundException {
         assertThat(dataInstanceResult).isEqualTo(dataInstance);
     }
 
-    private DataInstanceImpl createShortTextDataInstance(String value) {
-        DataDefinition dataDefinition = createDataDefinition();
+    private DataInstanceImpl createShortTextDataInstance(final String value) {
```

Comment:
Not really in this particular case.
I will remove them.
But if there was really a decision on this, the templates haven't been updated : https://github.com/bonitasoft/bonita-internal-tools/tree/master/codestyle